src/js/views/patients/shared/actions_views.js

```diff
@@ -22,11 +24,26 @@ const FormButton = View.extend({
   },
 });
 
+const DetailsTooltip = View.extend({
+  template: hbs`<span class="js-details">{{far "info-circle"}}</span>`,
+  ui: {
+    tooltip: '.js-details',
+  },
+  onRender() {
+    new Tooltip({
+      message: this.model.get('details'),
+      uiView: this,
+      ui: this.ui.tooltip,
```

Comment:
So I think this could probably just be `this.$el` and then no wrapping `<span>` / `ui` is necessary